aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -90,7 +90,7 @@ module aptos_framework::aptos_coin {
 
     /// Only callable in tests and testnets where the core resources account exists.
     /// Create new coins and deposit them into dst_addr's account.
-    public entry fun mint(
+public entry fun mint(
```

Comment:
Accidental space deletion?

Reply:
My IDE sucks. I am trying to switch. Which one you are using?

Reply:
VS Code

Reply:
@lightmark also I am using this extension: https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax

The jump-to-definition functionality is still lacking, but that's what the Vim extension is for: https://marketplace.visualstudio.com/items?itemName=vscodevim.vim